examples/audiotrack/main.go

```diff
@@ -132,14 +138,40 @@ func handlePublish(room *lksdk.Room) {
 	}
 }
 
-func handleSubscribe(track *webrtc.TrackRemote, targetChannels int) (*lkmedia.PCMRemoteTrack, *os.File) {
+func handleSubscribe(track *webrtc.TrackRemote, publication *lksdk.RemoteTrackPublication, targetChannels int) (*lkmedia.PCMRemoteTrack, *os.File) {
 	fileWriter, err := os.Create("test.mka")
 	if err != nil {
 		panic(err)
 	}
 
+	var decryptor lkmedia.Decryptor
+	passphrase := "helloworld"
```

Comment:
this seems to imply a static key for the lifetime of a subscription. 

I'm not sure if it's necessary here to implement key updates for go, just wanted to call out the constraints that this imposes.

I think that also in turn means there's currently no support for the key index defined in the encryption trailer in this SDK?

Reply:
Please check: [03b9830](https://github.com/livekit/server-sdk-go/pull/674/commits/03b983023048e7674b723bbea919706cc2761d94)

Reply:
> I think that also in turn means there's currently no support for the key index defined in the encryption trailer in this SDK?

Let me look into this

Reply:
Had a look, `kid` is not used for decrypting, the method directly expects the user to use the right key from the ring (bad design).  It is passed as parameter for encrypting though, so key index is technically supported, but totally managed by the user.